frontend/src/core/runtime/runtime.ts

```diff
@@ -52,14 +52,10 @@ export class RuntimeManager {
       }
     }
 
-    // Move over window level parameters to the WebSocket URL
```

Comment:
@dmadisetti Was there a reason for stripping unknown query parameters? This breaks `mo.query_params()`.

Was this code too aggressively refactored, i.e. is there a codepath that relies on query parameter stripping?

Reply:
A little too conservative - with the refactor query params were not passed down at all. Thanks for the general fix